torch_geometric/nn/pool/__init__.py

```diff
@@ -11,6 +11,8 @@
 from .asap import ASAPooling
 from .pan_pool import PANPooling
 from .mem_pool import MemPooling
+from .glob import (global_max_pool, global_mean_pool, global_add_pool,
```

Comment:
Ah sorry, this is not what I meant. I meant:
* Move `global_add_pool`, `global_mean_pool`, and `global_max_pool` to `glob/glob.py` to pool/glob.py`.
* Move `glob/__init__.py` to `nn/glob.py` and keep the deprecations in there.

Reply:
The idea here is that we keep the implementations of `global_*_pool` as they were (since they are so heavily used).

Reply:
hmm sorry I don't fully understand the distinction between these. In both cases you can 
```
from torch_geometric.nn import global_max_pool
```
is it that you want to keep this?
```
from torch_geometric.nn.glob import global_max_pool
````

Reply:
Or you just want to clearly separate the deprecations?

Reply:
Yes, we only need to
deprecate the import of `torch_geometric.nn.glob`.